docs/content/contributing/Troubleshooting.md

```diff
@@ -0,0 +1,20 @@
+# Troubleshooting
+
+## Configuration cache opt-out
+
+Experimental configuration cache support enabled by default for local work. 
+If something goes wrong with it and you have no time to fix it, you can add:
+
+`~/.gradle/gradle.properties`
+
+```properties
+org.gradle.unsafe.configuration-cache=false
+```
+
+**Why not opt-in?**
+
+Avito project has configuration cache disabled, just not all issues fixed for now.  
```

Comment:
```suggestion
Avito project has disabled configuration cache, just not all issues fixed for now.  
```
?

Reply:
i think both forms pretty common
really not worth another CI runs)

Reply:
Hm, never seen it